### ISSUE TITLE
Visual artifacts stay on screen when using "game pause" while on the white preview of a ride.

### ISSUE THREAD

**Issue by frenchiveruti** [OpenRCT2/OpenRCT2#7795]
**OS:** Windows 10
**Version:** OpenRCT2, v0.2.0-149 (10f6059 on develop) provided by AppVeyor
Windows (x86-64)
**Commit/Build:** 10f6059 

- [ ] Reproducible in RCT2 (vanilla)? Didn't check
- [ ] Specific to multiplayer?

**Steps to reproduce:**
Steps are clearer in the video provided

1. Select a ride, premade, with decoration. (I used an specific ride for the video.)
2. Move it until its "white preview" is visible on the screen.
3. **PAUSE the game**
4. Move the mouse or even stop placing the thing.
5. Success, virtual artifacts appear.

**Dump file**

_Again, there's no information around the wiki on how to generate this. At least not the "correct" or "most useful" way._

**Screenshots / Video:**

https://youtu.be/2pC5rx0Pr4Y

**Save game:**
Not save-game specific.

**Comment by Broxzier** [comment]
That the track preview stays on the same spot is indeed an original bug, but that the scenery pieces stay behind as ghosts when closing the construction window is not.

**Comment by frenchiveruti** [comment]
Yes, I add up to what @broxzier said, the scenery pieces staying there in the original game doesn't happen, the track staying visible in place when pausing is actually a feature, it allows you to rotate the view to see how it fit.
